### ISSUE TITLE
check_model issue

### ISSUE THREAD

**Issue by juadiegaitan** [easystats/performance#285]
Hi, I am running the check_model function in RStudio (R version 4.0.5) for a glm model but keep getting the following error: 
Error in 1:label.n : NA/NaN argument
I can't find any issue in the model nor information online about potential cause. Any similar experience or ideas to solve this issue?
Thanks

**Comment by bwiernik** [comment]
Can you give a small reproducible example?

**Comment by strengejacke** [comment]
And the session info - are all packages up to date?

**Comment by craddm** [comment]
Same issue here.

``` r
library(performance)
library(lme4)
#> Loading required package: Matrix
sleepstud_fac <- sleepstudy
sleepstud_fac$Days <- as.factor(sleepstud_fac$Days)
sleep_lmer <- lmer(Reaction ~ Days + (1|Subject), data = sleepstud_fac)
check_model(sleep_lmer)
#> Error in 1:label.n: NA/NaN argument
sessionInfo()
#> R version 4.0.3 (2020-10-10)
#> Platform: x86_64-pc-linux-gnu (64-bit)
#> Running under: Ubuntu 20.04.2 LTS
#> 
#> Matrix products: default
#> BLAS:   /usr/lib/x86_64-linux-gnu/atlas/libblas.so.3.10.3
#> LAPACK: /usr/lib/x86_64-linux-gnu/atlas/liblapack.so.3.10.3
#> 
#> locale:
#>  [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       
#>  [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   
#>  [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          
#> [10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   
#> 
#> attached base packages:
#> [1] stats     graphics  grDevices utils     datasets  methods   base     
#> 
#> other attached packages:
#> [1] lme4_1.1-26       Matrix_1.2-18     performance_0.7.1
#> 
#> loaded via a namespace (and not attached):
#>  [1] statmod_1.4.36     tidyselect_1.1.1   xfun_0.22          purrr_0.3.4       
#>  [5] splines_4.0.3      lattice_0.20-41    parameters_0.13.0  colorspace_2.0-1  
#>  [9] vctrs_0.3.8        generics_0.1.0     htmltools_0.5.1.1  yaml_2.2.1        
#> [13] utf8_1.2.1         rlang_0.4.11       nloptr_1.2.2.2     pillar_1.6.0      
#> [17] glue_1.4.2         withr_2.4.2        DBI_1.1.1          effectsize_0.4.4-1
#> [21] plyr_1.8.6         emmeans_1.6.0      lifecycle_1.0.0    stringr_1.4.0     
#> [25] munsell_0.5.0      gtable_0.3.0       bayestestR_0.9.0   mvtnorm_1.1-1     
#> [29] evaluate_0.14      knitr_1.33         fansi_0.4.2        highr_0.9         
#> [33] Rcpp_1.0.6         xtable_1.8-4       scales_1.1.1       see_0.6.3         
#> [37] fs_1.5.0           ggplot2_3.3.3      digest_0.6.27      stringi_1.6.1     
#> [41] ggrepel_0.9.1      insight_0.14.0     dplyr_1.0.6        grid_4.0.3        
#> [45] cli_2.5.0          tools_4.0.3        magrittr_2.0.1     tibble_3.1.1      
#> [49] crayon_1.4.1       pkgconfig_2.0.3    MASS_7.3-53        ellipsis_0.3.2    
#> [53] ggridges_0.5.3     estimability_1.3   reprex_2.0.0       assertthat_0.2.1  
#> [57] minqa_1.2.4        rmarkdown_2.8      rstudioapi_0.13    R6_2.5.0          
#> [61] boot_1.3-25        nlme_3.1-149       compiler_4.0.3
```

<sup>Created on 2021-05-12 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by IndrajeetPatil** [comment]
The issue should be fixed in `parameters 0.14.0`, which is on CRAN.

Please re-open with a reproducible example if the issue persists even after updating to the latest `parameters` version.
